sections/collage.liquid

```diff
@@ -59,14 +59,21 @@
                 extend_height: true
               %}
             {%- when 'collection' -%}
+              {%- liquid
+                  assign wrapper_class = section.settings.card_styles
+                  if section.settings.card_styles == 'none' 
+                    assign wrapper_class = 'collection-card-wrapper'
+                  endif
+               -%}
               {% render 'card-collection',
                 card_collection: block.settings.collection,
                 media_aspect_ratio: 'adapt',
                 columns: 2,
-                extend_height: true
+                extend_height: true,
+                wrapper_class: wrapper_class
```

Comment:
If we only have the choice of `product-card-wrapper` and `product-card`, would it make sense for this parameter to be a boolean? If we have plans for more options, just ignore my question :D

Like I said in my previous review, I noticed we're using `product-card-wrapper` hardcoded all over this file except for, where we use the setting value.

If we're okay with having it hardcoded in `card-collection` too, we could do something like this there:
```liquid
{% if override_card_style %}product-card-wrapper{% else %}collection-card-wrapper{% endif %}
```

Reply:
The reason I wanted to write it like this is because eventually we may want to add other card types as options 🤔 Does that make sense? So I can update the hardcoded `product-card-wrapper` class

Reply:
Ah, that's okay. Let's keep it as is :)